.packit.yaml

```diff
@@ -43,9 +45,22 @@ jobs:
     notifications: &copr_build_failure_notification
       failure_comment:
         message: "Ephemeral COPR build failed. @containers/packit-build please check."
+    targets: &fedora_copr_targets
+      - fedora-development-x86_64
+      - fedora-development-aarch64
+      - fedora-latest-x86_64
+      - fedora-latest-aarch64
+      - fedora-latest-stable-x86_64
+      - fedora-latest-stable-aarch64
+      - fedora-40-x86_64
+      - fedora-40-aarch64
```

Comment:
Any reason to not use fedora-all here?

Reply:
I mean even if f39 is disabled in the copr it should still build fine here in the tests but I guess overall it doesn't matter and I am fine to remove it.

Reply:
mainly because of reusing `fedora_copr_targets` in the test job below where fedora-39 would fail. I'm ok to have fedora-all and only use non-fedora-39 in the tests.